ros/src/tl_detector/tl_detector.py

```diff
@@ -236,13 +231,16 @@ def process_ground_truth_lights(self):
                     if ((ntl_wp - position) < stop_distance):
                         stop_distance = (ntl_wp - position)	
                         stop_line = position
+
         if stop_line > 0:
-            rospy.loginfo('Light state: %s',ntl_state)
-            rospy.loginfo('Stop line: %f',stop_line)			
-            return stop_line, ntl_state 
+            if TLC_ENABLED:
+               ntl_state = self.get_light_state(light)
```

Comment:
Not an issue, but this variable `light` is probably no longer needed.  I think Udacity put this in originally because people were trying to use camera transformation matrices to crop a piece of the camera image that should correspond to a lights world position coordinates before classifying, but it caused too much trouble from the differences between the simulator and Carla's cameras so they decided to let everyone just go with full image neural net detection instead where the light world position is not needed for the classifier anymore.

Reply:
Will hold this variable change until we know how the classifier is implemented.do you agree?

Reply:
Sure, that’s fine.